File: easy/2.py
N = int(input())
x = list(map(int, input().split()))

ans = 10 ** 10
max_num = max(x)
for i in range(1, max_num+1):
    tmp = 0
    for xi in x:
        tmp += (i-xi)**2

    if ans > tmp:
        ans = tmp
print(ans)

File: midium/10/2.go
package main

import "fmt"



func main() {
	var s string
	fmt.Scan(&s)
	b_count:= 0
	ans := 0
	for _, si := range s {
		if si == 'B'{
			b_count++
		}
		if si == 'W'{
			ans += b_count
		}
	}
	fmt.Println(ans)
}